ynput/ayon-core
# Hiero: loader is offering even on unsaved project file

## Changelog Description
Unsaved Hiero workfile has now offered product versions in Loader. 

## Additional info
Adjusted loop to handle empty current sequence by appending an empty list if no tracks are present.


## Testing notes:
1. open Hiero and make sure you are in `Untitled` project file
2. open Loader and see that it looks normal and product verisons are offered.
